arcgis-ios-sdk-samples/AppDelegate.swift

```diff
@@ -169,9 +172,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UISplitViewControllerDele
 }
 
 extension UIColor {
-    class var primaryBlue: UIColor { return #colorLiteral(red: 0, green: 0.475, blue: 0.757, alpha: 1) }
-    class var secondaryBlue: UIColor { return #colorLiteral(red: 0, green: 0.368, blue: 0.584, alpha: 1) }
-    class var backgroundGray: UIColor { return #colorLiteral(red: 0.973, green: 0.973, blue: 0.973, alpha: 1) }
-    class var primaryTextColor: UIColor { return #colorLiteral(red: 0.196, green: 0.196, blue: 0.196, alpha: 1) }
-    class var secondaryTextColor: UIColor { return #colorLiteral(red: 0.349, green: 0.349, blue: 0.349, alpha: 1) }
+    // Also used for global tint color.
+    class var primaryThemeColor: UIColor { return UIColor(named: "primaryThemeColor")! }
+    class var secondaryThemeColor: UIColor { return UIColor(named: "secondaryThemeColor")! }
```

Comment:
I will remove not used colors from code.

However, I'd like to keep them in asset catalog. They are copied from Esri's branding guide, and I think it would be good to have a few "safe" colors to use in case we need them in the future, rather than using arbitrary custom or system colors. 😳 

If need to remove, I'll still keep 1 translucent background gray for future dark mode adaption, even if it is not in use right now.

Reply:
It is generally a best practice to not include unused code or assets in a repo as it unnecessarily as it can have negative consequences, such as increasing build time and the size of the app. While leaving these colors in would have a negligible impact on build time and size, there are other consequences. The colors in the branding guide could change between now and when the unused colors may be used, potentially leading to using the wrong color. And sometimes plans change, meaning that the colors could end up not being used at all.

It is easy enough to add a color asset to the asset catalog. I would strongly suggest including only the color assets that are currently being used and adding additional colors only when they become needed.